.github/workflows/build.yml

```
@@ -21,6 +21,8 @@ jobs:
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: |
+            8.0.x
+            7.0.x
             6.0.x
             3.1.x
```

```suggestion
            8.0.x
            6.0.x
```
Let's remove versions 7.0 and 3.1, as we don't use them.